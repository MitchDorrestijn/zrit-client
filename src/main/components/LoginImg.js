/**
 * React related imports
 */
import React from 'react';

/**
 * Other imports
 */
import HomeImg from '../assets/img/home.jpg';

/**
 * This component renders the image on the left side of login page
 */
const LoginImg = () => {
  return (
    <div className="grandParentContaniner">
      <div className="parentContainer">
        <img src={HomeImg} className="img-fluid" alt="zorgrit introimage"/>
      </div>
    </div>
  );
}

export default LoginImg;
