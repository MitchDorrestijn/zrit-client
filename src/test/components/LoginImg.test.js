import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import LoginImg from '../../main/components/LoginImg';

const wrapper = shallow(<LoginImg />);

test('<LoginImg /> should match snapshot', () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
});

test('<LoginImg /> should render <div></div> two times', () => {
    expect(wrapper.find('div').length).toBe(2);
});

test('<LoginImg /> should render one <img /> tag', () => {
    expect(wrapper.find('img').length).toBe(1);
});

test('<img /> tag should have the class .img-fluid', () => {
    expect(wrapper.find('img').hasClass('img-fluid')).toBe(true);
});
