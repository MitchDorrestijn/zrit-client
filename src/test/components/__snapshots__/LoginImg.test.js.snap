// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm component should match snapshot 1`] = `
<div
  className="grandParentContaniner"
>
  <div
    className="parentContainer"
  >
    <img
      alt=""
      className="img-fluid"
      src="home.jpg"
    />
  </div>
</div>
`;
